doubles/pytest_plugin.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from doubles.lifecycle import teardown, verify
+
+
+try:
+    @pytest.hookimpl(hookwrapper=True)
```

Comment:
Nice!

I wonder if there is a better way than those ugly `try`/`except` (I'd at least use `except Exception` so we don't catch things like `^C` and `SystemExit`).

Is there something we could try to import and then catch an `ImportError`? Or maybe something we could introspect? Doing this would also be more readable/explicit than `try`/`except`.